ServerMonitoringSystem.MessageProcessor/Services/MessageProcessor.cs

```diff
@@ -0,0 +1,43 @@
+﻿using MessagingLibrary.Interfaces;
+using MessagingLibrary.RabbitMq;
+using ServerMonitoringSystem.MessageProcessor.Persistence;
+using ServerMonitoringSystem.MessageProcessor.Services;
+using ServerMonitoringSystem.MessageProcessor.Services.Interfaces;
+using ServerMonitoringSystem.Shared.Domain;
+using System.Text.Json;
+
+public class MessageProcessor : IMessageProcessor
+{
+    private readonly IStatisticsRepository _repository;
+    private readonly IMessageConsumer _consumer;
+    private readonly IAnomalyDetector _anomalyDetector;
+    private readonly ISignalRAlertSender _notifier;
+
+    public MessageProcessor(IStatisticsRepository repository, string rabbitMqHost, string exchange, string queue, string routingKey,
```

Comment:
we should be injecting `IMessageConsumer` instead of instantiating it from within the constructor and passing its configs here.